resources/public/api/conf/6.0/schemas/retrieve_foreign_property/def2/retrieve_foreign_property.json

```diff
@@ -756,83 +756,6 @@
             "additionalProperties": false
           }
         },
```

Comment:
Looks good but can this comma be removed, thanks. When I validate the JSON using https://jsonlint.com/, it states the JSON is invalid. It will work locally but there will be publishing issues in the environment which have more robust checks.

Reply:
Also, it appears there is some inconsistency in the specifications. According to the business spec, adjustments is intended to be an array, but the tech spec states it can either be an array or an object which I think is wrong as it is always an array as per the downstream spec. Currently, in the code and OAS, it is implemented as an object. This works because of the presence of the countryLevelDetail layer.

However, from 25-26 onwards, the Foreign FHL and Foreign Non-FHL property types will no longer be valid. This means we probably need to decide whether adjustments should be strictly an array or remain as an object. It was previously implemented as an object to standardise the structure and leverage the countryLevelDetail for array entries.

Since adjustments will consistently represent array entries from 25-26 onwards, I think we need decide whether to retain the countryLevelDetail layer or directly make adjustments an array. The business spec seems to favour making adjustments an array, but that needs clarification. Can ignore this for now until it is confirmed.